src/java/main/org/apache/zookeeper/server/persistence/FileTxnLog.java

```diff
@@ -199,14 +199,16 @@ public synchronized boolean append(TxnHeader hdr, Record txn)
                 LOG.warn("Current zxid " + hdr.getZxid()
```

Comment:
@maoling my vim editor will remove the tailing space automatically, that's why there is so many changes, I think it's better to format it without tailing space.